.pre-commit-config.yaml

```diff
@@ -1,12 +1,24 @@
 exclude: ^user_docs/(?!en/).*$
 
+# https://pre-commit.ci/
+# Configuration for Continuous Integration service
+ci:
+  # Can't run Windows scripts on Linux
+  # These also do not apply fixes, only run tests
+  skip: [scons-source, checkPot, unitTest, licenseCheck]
+  submodules: true
+  autoupdate_commit_msg: "Pre-commit autoupdate"
+  autofix_commit_msg: "Pre-commit autofix"
```

Comment:
```suggestion
  autoupdate_commit_msg: "Pre-commit automatic update"
  autofix_commit_msg: "Pre-commit automatic fix"
```

Reply:
Could you open a small PR to fix this up. Maybe auto-fix and auto-update instead?